### DISCUSSION TITLE
How to open a link in a new tab/window

### DISCUSSION THREAD

**Discussion by CptMonac** [chrisgurney/obsidian-note-toolbar#81] [General]
Hi Chris,

Echoing what others have commented on here, love love love the plugin and the cadence of new features and goodies. I'm using the toolbar with the Daily Notes plugin and have added links to other pages in my vault on the toolbar. I can't seem to figure out how to get these pages to open in a new tab or window when I click on them though - the pages always replace the current active page and ideally I'd like to have them side-by-side. Is this functionality currently supported?

**Comment by chrisgurney** [comment]
@CptMonac What I believe is happening here is that I had to change toolbar items from links to `spans` (that act as links), in order to fix  #19 which is particular to Android.

What I may end up doing -- since you're the second person to request this, similar to #80 -- is conditionally using `spans` _only if_ the current platform is Android... but this will take some testing to make sure I don't introduce any major regressions.

**Comment by isidarbest** [comment]
Hi there, I am adding my vote to adding this functionality too :) or having osme way to choose the default behaviour (if new tab or replace existing one)

**Comment by chrisgurney** [comment]
Closing this discussion, as I've moved it into issue #91
